pkg/broker/ttl.go

```diff
@@ -23,7 +23,9 @@ import (
 const (
 	// V02TTLAttribute is the name of the CloudEvents 0.2 extension attribute used to store the
 	// Broker's TTL (number of times a single event can reply through a Broker continuously).
-	V02TTLAttribute = "knativebrokerttl"
```

Comment:
just changing the case is not really a fix because it will only work if using the golang net stack.  There is code that normalizes the casing to lower and then looks for it, for some reason that code was broken.

Reply:
see updated commit